topic_list:
  - {
      topic: "cam_back/image_rect_color",
      expected_hz: 10.0,
      tolerance: 1.0
    }
  
  - {
      topic: "cam_front/image_rect_color",
      expected_hz: 10.0,
      tolerance: 1.0
    }

  - {
      topic: "cam_left/image_rect_color",
      expected_hz: 10.0,
      tolerance: 1.0
    }

  - {
      topic: "cam_right/image_rect_color",
      expected_hz: 10.0,
      tolerance: 1.0
    }

  - {
      topic: "imu/data",
      expected_hz: 500.0, 
      tolerance: 5.0
    }

  - {
      topic: "horiz/os_cloud_node/points",
      expected_hz: 20.0,
      tolerance: 2.0
    }

  - {
      topic: "vert/os_cloud_node/points",
      expected_hz: 20.0,
      tolerance: 2.0
    }

  - {
      topic: "scan",
      expected_hz: 10.0,
      tolerance: 2.0
    }

  - {
      topic: "pico_flexx_left/points",
      expected_hz: 5.0,
      tolerance: 1.0
    }

  - {
      topic: "pico_flexx_right/points",
      expected_hz: 5.0,
      tolerance: 1.0
    }

  - {
      topic: "lio_sam/mapping/odometry",
      expected_hz: 5.0,
      tolerance: 3.0
    }

  - {
      topic: "merged_map",
      expected_hz: 1.0,
      tolerance: 0.5
    }

  - {
      topic: "octomap_in",
      expected_hz: 12.0,
      tolerance: 8.0
    }

  - {
      topic: "scan_plan_node/path_out",
      expected_hz: 4.0,
      tolerance: 3.0
    }
